.github/workflows/ci-cd-github-actions.yml

```diff
@@ -21,3 +21,9 @@ jobs:
       - run: npm run lint
       - run: npm test -- --watch=false --browsers=ChromeHeadless
       - run: npm run build
+      - uses: w9jds/firebase-action@master
+        with:
+          args: deploy --only hosting
+        env:
+          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
```

Comment:
Where did you put it, can you explain? Looking cool!

Reply:
<img width="1164" alt="image" src="https://user-images.githubusercontent.com/17585880/195601502-76e937ef-20b9-4d31-a113-66f954487ad1.png">
 under the repo, when I used your git account.
 
 Have updated the FIREBASE_TOKEN btw?